src/Engine/GameEvents/TurnEndEvent.java

```diff
@@ -27,6 +27,11 @@ public GameEventQueue sendToListener(GameEventListener listener)
     return listener.receiveTurnEndEvent(army, turn);
   }
 
+  @Override
+  public void performEvent(MapMaster map)
```

Comment:
Is there some reason `TurnEndEvent` needs to override `TurnInitEvent` at all, if we are just gonna override behavior to stop it? It sounds kinda sketchy from the word go.

Reply:
I know I had a reason, but I can't recall it anymore.
...I think I just didn't feel like copy/pasting the "do nothing" methods.
That seems like a "me" thing to do.

Reply:
Haha, well I hope we learned a valuable lesson today.